src/Symfony/Bridge/PhpUnit/Legacy/ExpectDeprecationTrait.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Legacy;
+
+/**
+ * @internal use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait instead.
+ */
+trait ExpectDeprecationTrait
```

Comment:
let's same it ExpectDeprecationTraitFor8_4 to not create any potential conflict when ppl will use autocompletion

Reply:
ExpectDeprecationTraitFor**V**8_4 to stay consistent